src/NuGet.Clients/NuGet.VisualStudio.Common/OutputConsoleLogger.cs

```diff
@@ -104,8 +105,13 @@ public void End()
                 await _outputConsole.WriteLineAsync(Resources.Finished);
                 await _outputConsole.WriteLineAsync(string.Empty);
 
-                // Give the error list focus
-                await _errorList.Value.BringToFrontIfSettingsPermitAsync();
+                if (_errorsReported)
+                {
+                    // Give the error list focus
+                    await _errorList.Value.BringToFrontIfSettingsPermitAsync();
+
+                    _errorsReported = false;
```

Comment:
Might as well set this before calling `_errorList.Value.BringToFrontIfSettingsPermitAsync()` just in case it throws for some reason?

Reply:
I put it here for the reverse. If something happens that we didn't show the error list, keep it in a "pending error list focus" state for the next round.
Also... if that throws, there's likely going to be way worse behavior, guessing a crash.